tls/s2n_connection.c

```diff
@@ -249,28 +249,12 @@ static int s2n_connection_zero(struct s2n_connection *conn, int mode, struct s2n
     /* Zero the whole connection structure */
     memset_check(conn, 0, sizeof(struct s2n_connection));
 
-    conn->send = NULL;
```

Comment:
Why are you removing all these lines? What does this have to do with the handshake arrays?

Reply:
I can remove that change if it distracts too much. In a previous version of this change I needed to remove one of those lines, so I decided to clean up the rest while i was there.

Anything that sets a variable to 0/NULL is unnecessary because the previous line does "memset_check(conn, 0, sizeof(struct s2n_connection));", initializing the entire structure's memory to 0/NULL.